### PR TITLE
Added Scss and sctipts

### PR DESCRIPTION
**Pull request by Olwiba**

### CODE REVIEW

**Comment by vaishnavi-iyer** [comment]
not passing Travis ... pull master and then need to create another pull request

**Comment by mixmix** [comment]
Heads up, you don't need to close a pull request... 
You can pull `master` into branch `ollie` and when you push it up to github again, the pull request will get updated

File: client/scss/nav.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
+@import 'https://fonts.googleapis.com/css?family=Press+Start+2P';
```
Comment (mixmix) [inline client/scss/nav.scss:1]: here is an example of commenting on some code.
I clicked on "Files changed" and then clicked on the line I wanted to comment on. 

You can discuss things you're not sure about in this way

Comment (mixmix) [inline client/scss/nav.scss:1]: e.g. I might say "do we want to import fonts from Google ... how about we do X 
OR  awesome :fire:

**Pull request closed without merging**